### ISSUE TITLE
Is there any possibility to visualize the SGBClassifier results?

### ISSUE THREAD

**Issue by jmq19950824** [scikit-learn-contrib/sklearn-pandas#196]
I was trying to implement a classification problem by SGDClassifier and used the following codes:

SGDClassifier(penalty='elasticnet',random_state=seed,shuffle=False,loss='log')

i.e., I use a Logistic Regression for this classification problem and deploy both l1 and l2 regularized terms (elasticnet) to this model. Now the problem is I want to visualize the result like the enet_path in the sklearn (please see the following picture)
![image](https://user-images.githubusercontent.com/39890732/56554838-9c7e5b00-65c5-11e9-9d7d-4c8c7921e675.png)

As far as I know, the enet_path is for ElasticNet regression, not the classification problem that I want. So is there any method to realize this problem?

By the way, I have tried glmnet package which can conveniently solve this problem in R. Unfortunately, this package could not install in Python under my Win10 environment (need Linux).
